### DISCUSSION TITLE
Question about working with gltf files?

### DISCUSSION THREAD

**Discussion by denisemendez007** [AR-js-org/AR.js#423] [Q&A]
I am playing with gltf's found online through sketchfab and other free asset sites.  I am finding not all gltfs work in their ability to be displayed on the phone above the marker, but others don't.  Are there common issues with why some gltf's will work and others will not?  Polygon count, scale etc?  How would one troubleshoot why an object isn't being displayed when others work fine?

I have a glitch setup where I am playing with different files.  You can see it here https://glitch.com/edit/#!/north-resonant-marquis?path=index.html%3A18%3A64.  There is an asset called scene.gltf and another called rigged hands that I just can't get to display.  But the other seem to work fine.

I am following along with MindAr's example (https://hiukim.github.io/mind-ar-js-doc/quick-start/assets).

**Comment by nikoyanakiev** [comment]
Hi, maybe you can compare what this gltf viewer returns as it analyses the files: [https://gltf-viewer.donmccurdy.com/](https://gltf-viewer.donmccurdy.com/). In my case was similar - I've downloaded a very basic gltf file from sketchfab, which I can't display, but later scanned and generated a glb from poycam, which is integratable perfectly with the ar.js examples.
